packages/3id-did-resolver/src/index.ts

```diff
@@ -109,9 +115,9 @@ function extractMetadata(
   const { timestamp: updated, cid: versionId } = lastAnchorOrGenesisEntry(requestedVersionState.log)
 
   const { timestamp: nextUpdate, cid: nextVersionId } =
-    latestVersionState.log.find(
-      ({ timestamp }) => timestamp > updated || (!updated && timestamp)
-    ) || {}
+    latestVersionState.log.find(({ timestamp }) => {
+      return (timestamp && updated && timestamp > updated) || (!updated && timestamp)
```

Comment:
It's _ok_ like this, but might be more readable as `return (timestamp && ((updated && timestamp > updated) || !updated))`

Reply:
As it is currently written, it just takes a little bit longer to understand.

Reply:
Yup. Changed that to the suggested expression.